tests/dragonfly/seeder/__init__.py

```diff
@@ -137,6 +138,8 @@ def __init__(
         data_size=100,
         collection_size=None,
         types: typing.Optional[typing.List[str]] = None,
+        huge_value_percentage=0,
```

Comment:
For now I will keep a flat probability for each key in the key target to contain a `huge value`